torch/_torch_docs.py

```diff
@@ -619,6 +619,105 @@ def merge_dicts(*dicts):
     True
 """)
 
+add_docstr(torch.all,
```

Comment:
I think these need entries in torch.rst, too?

Reply:
Done